doc/tutorials/introduction/desktop_java/java_dev_intro.markdown

```diff
@@ -274,7 +274,8 @@ class DetectFaceDemo {
 
     // Create a face detector from the cascade file in the resources
     // directory.
-    CascadeClassifier faceDetector = new CascadeClassifier(getClass().getResource("/lbpcascade_frontalface.xml").getPath());
+    File file = new File(getClass().getResource("/lbpcascade_frontalface.xml").getFile());
+    CascadeClassifier faceDetector = new CascadeClassifier(file);
     Mat image = Imgcodecs.imread(getClass().getResource("/lena.png").getPath());
```

Comment:
I believe, `imread()` call has exactly the same problem (with other similar hits in the codebase).

---

Actually "File" approach may not work well with packaged applications (.jar/.war or .apk on Android).

I believe robust way is passing `InputStream` via `.getResourceAsStream()` / `.openRawResource()` calls.
But it requires extra design work (proper "stream" reader on C++ side (`std::istream`) and proper wrappers for Java->C++ streams in bindings).